models/table_200x100x76/model.sdf

```diff
@@ -5,15 +5,15 @@
       <collision name="1">
         <geometry>
           <box>
-            <size>2.0 1.0 0.02</size>
+            <size>2.05 1.05 0.02</size>
```

Comment:
This table is also in `rosebv`. We are probably not using that anymore. But stupid to break stuff if it is still there. Also the name contains the dimensions of this model, so rename please.

Reply:
rosebv is no more indeed. But plz copy to a new `models/table_205x105x76/model.sdf`